vivo/runtime.properties.jr

```diff
@@ -0,0 +1,342 @@
+# -----------------------------------------------------------------------------
```

Comment:
Is there a reason for the inclusion of this file in the pull-request?

Reply:
Nope. We'll remove.  We are also looking at a few other modifications to this, (mostly adding proper volumes) so you might delay, and wait for a few additional changes.